ros2bag/ros2bag/verb/play.py

```diff
@@ -52,14 +52,22 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             '--topics', type=str, default=[], nargs='+',
             help='topics to replay, separated by space. If none specified, all topics will be '
                  'replayed.')
+        parser.add_argument(
+            '--services', type=str, default=[], nargs='+',
+            help='services to replay, separated by space. At least one service needs to be '
```

Comment:
> I think we need to keep the same style for the description, see the --topics above.
> It seems unnecessary to use "At least one" because the help information shows "--topics TOPICS [TOPICS ...]" that users should know this style usage.

Yes.  Update it. 

> The "If none specified" description might include the "--topics" itself.

Yes. Like "--topics", if users don't specify "--service", all recorded service requests will be played.